### DISCUSSION TITLE
[FEATURE]: why we need conductor,not jbpm ,not activiti

### DISCUSSION THREAD

**Discussion by chrishyc** [Netflix/conductor#3132] [General]
what scenarios are suitable for conductor and why we need conductor not jbpm, not activiti?

**Comment by jxu-nflx** [comment]
Hello @chrishyc you could check this doc to see whether Conductor is a good fit for you: https://conductor.netflix.com/gettingstarted/intro.html

**Comment by boney9** [comment]
Also @chrishyc - we have some write ups here as well: https://orkes.io/what-is-conductor/ and https://orkes.io/content/blog
If you'd like to meet to discuss more - please do let me know.
